common/Cargo.toml

```diff
@@ -11,5 +11,8 @@ hex = "0.4.3"
 ssz-rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "d09f55b4f8554491e3431e01af1c32347a8781cd" }
 ethers-core = "2.0.8"
 thiserror = "1.0.37"
+candid = "0.9.1"
```

Comment:
remove candid and use [this](https://docs.rs/ic-cdk/latest/ic_cdk/api/management_canister/http_request/fn.http_request.html)

Reply:
done